river/View.zig

```diff
@@ -557,6 +556,28 @@ pub fn attachAfter(view: *Self, pending_state: *Output.PendingState, n: usize) v
     it.current.prev.?.insert(&view.pending_wm_stack_link);
 }
 
+/// Attach above or below the currently focused view
+pub fn attachRelative(view: *Self, pending_state: *Output.PendingState, mode: AttachRelativeMode) void {
+    var focus_stack_it = pending_state.focus_stack.iterator(.forward);
+    const focused_view = focus_stack_it.next();
```

Comment:
I'm wondering if there is a case where the first view in the pending focus stack is not visible with the pending tags of the output. I don't think there is, so it would be good to add an assert that `focus_view.pending.tags & pending_state.tags != 0` here as we rely on that fact.

Reply:
This assertion fails when opening the first view on tag with other tags already in use.

Reply:
Ah yeah, makes sense. It would make me happy if that edge case was more visible in this code to stop me from adding a bad assertion or otherwise breaking things in the future. Maybe it just needs a comment to prevent future regressions.